src/elements/mod.rs

```diff
@@ -257,6 +257,18 @@ impl From<io::Error> for Error {
 	}
 }
 
+// These are emitted by section parsers, such as `parse_names` and `parse_reloc`.
+impl From<(Vec<(usize, Error)>, Module)> for Error {
+	fn from(err: (Vec<(usize, Error)>, Module)) -> Self {
+		let ret = err.0.iter()
+			.fold(
+				String::new(),
+				|acc, item| acc + &format!("In section {}: {}\n", item.0, item.1)
```

Comment:
I think this will be quadratic reallocation

Just use `acc.push_str(format!(...))`

Reply:
Thanks! Changed it.